src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
+import copy
```

Comment:
![image](https://github.com/microsoft/promptflow/assets/47586720/4dc5d215-96a4-4fc9-a862-703f250f529b)
Can we use / instead of \ in result?